patients/patients.go

```diff
@@ -94,6 +94,13 @@ func (p Patient) IsCustodial() bool {
 	return p.Permissions != nil && p.Permissions.Custodian != nil
 }
 
+type PatientDeletion struct {
```

Comment:
[nit] Its up to you what you want to name this, but, as commented before, consider just `Deletion` since it is in the `patients` package. I see you made some changes, but didn't get the others. Perhaps a case-insensitive search on "patientdeletions" and review for other changes? Entirely up to you, though.